app.js

```diff
@@ -0,0 +1,123 @@
+//<=======================create template=========================>
+
+const getTemplateArr = (size, arr = [], counter = 0) => {
+  for (var i = 0; i < size; i++) {
+    arr[i] = [];
+    for (var j = 0; j < size; j++, counter++) {
+      arr[i][j] = counter;
+    }
+  }
+  return arr;
+};
+
+console.table(getTemplateArr(10));
+
+//<=======================create Bomba=========================>
+
+const checkBomba = arr => {
+  return arr.map(n => {
+    let res = (n % 3 ? '' : 'Super') + (n % 5 ? '' : 'Buper') || n;
+    return res === 'SuperBuper' ? (res = 'BOOM') : res;
+  });
+};
+
+const getBombaArr = size => {
+  return getTemplateArr(size).map(n => checkBomba(n));
```

Comment:
Try this `return getTemplateArr(size).map(checkBomba)`;

Reply:
Yeah, i forgot to change